### ISSUE TITLE
Warning introduced by version 0.42

### ISSUE THREAD

**Issue by rikardn** [mgedmin/check-manifest#119]
When using version 0.42 of check-manifest I get a warning that version 0.41 didn't give:

I run:

```
check-manifest -v /myproject
```

The warning:

```
..../lib/python3.7/site-packages/setuptools/distutils_patch.py:26: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  "Distutils was imported before Setuptools. This usage is discouraged "
```

**Comment by MkLHX** [comment]
https://github.com/mgedmin/check-manifest/pull/117

i think just be patient and @mgedmin  accept the pull request ;-)

**Comment by rikardn** [comment]
Thanks! Sorry, I didn't see the PR. These guys are fast, I was thinking, solving my issue 7 days before I even posted it :smile:

**Comment by mgedmin** [comment]
Thank you for the heads up!  Fixed by #117.

(Apologies for a late reply, I was on vacation and was pretending computers didn't exist.)

**Comment by rikardn** [comment]
Thanks for fixing this! I tested the update and the warning is now gone.
